### DISCUSSION TITLE
https://v3.umijs.org/zh-CN/plugins/plugin-qiankun教程中的Examples示例代码在哪个分支呢？

### DISCUSSION THREAD

**Discussion by kgloveyou** [umijs/umi#12139] [Q&A]
https://v3.umijs.org/zh-CN/plugins/plugin-qiankun教程中的Examples示例代码在哪个分支呢？

**Comment by fz6m** [comment]
umi 3 qiankun 插件源码位置：https://github.com/umijs/plugins/tree/master/packages/plugin-qiankun
